### PR TITLE
Fix Autoform Submit Problem

### PR DESCRIPTION
**Pull request by cpravetz**
Fixes problem with callbacks being lost because args passed from autoform include a validation object while collection-hooks assumes only two args.

### CODE REVIEW

**Comment by jankapunkt** [comment]
@cpravetz thanks a lot this pull request. Can you please add a bit more info, for which AutoForm method or hook this fix is?

**Comment by cpravetz** [comment]
Sorry - the patch addresses issue #1710 in the meteor-autoform repo. The callback passed during submit gets lost when collection-hooks expects two arguments and autoform passes three - so actions after the actual collection submit happens are never called.

**Comment by jankapunkt** [comment]
Related: https://github.com/Meteor-Community-Packages/meteor-autoform/issues/1710

File: insert.js [hunk 1]
```diff
@@ -4,8 +4,13 @@ import { CollectionHooks } from './collection-hooks'
...
   const ctx = { context: this, _super, args }
-  let [doc, callback] = args
-  const async = typeof callback === 'function'
+  let doc = args[0]
+  let callback
+  if (typeof args[args.length -1] === 'function')  {
+    callback = args[args.length - 1]
+  }
+  
+const async = typeof callback === 'function'
```
Comment (jankapunkt) [inline insert.js:13]: Please indent this line by two spaces.

**Review by jankapunkt — CHANGES_REQUESTED** [review]
Just a minor fix then we can merge

File: insert.js (hunk 1 above)
Comment (cpravetz) [reply insert.js]: Done.

**Review by jankapunkt — APPROVED** [review]

**Comment by jankapunkt** [comment]
@cpravetz all good
@StorytellerCZ this can be merged and published. Can you do that?

**Comment by copleykj** [comment]
@jankapunkt you should have the ability to do that if you wanted to.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]
Please add a changelog entry and then I will merge and publish a patch release.

**Review by StorytellerCZ — APPROVED** [review]

**Comment by StorytellerCZ** [comment]
Published in matb33:collection-hooks@1.1.3

**Pull request merged**